srcdeps-core/src/main/java/org/srcdeps/core/config/ScmRepository.java

```diff
@@ -153,18 +167,29 @@ public Builder id(String id) {
             return this;
         }
 
-        public Builder maven(ScmRepositoryMaven.Builder maven) {
-            this.maven.init(maven);
+        public Builder include(String include) {
+            this.includes.add(include);
             return this;
         }
 
-        public Builder selector(String selector) {
-            this.selectors.add(selector);
+        public Builder includes(List<String> includes) {
+            this.includes.addAll(includes);
             return this;
         }
 
+        /**
+         * Kept for backwards compatibility, as {@code selectors} were renamed to {@code includes}
```

Comment:
+1 , I guess this completely answers my backwards compatibility concerns.

Reply:
:-)